src/main/scala/com/twitter/diffy/lifter/JsonLifter.scala

```diff
@@ -25,7 +25,7 @@ object JsonLifter {
     case JsonToken.START_ARRAY        =>
       node.elements.toSeq.map {
         element => lift(element)
-      }  sortBy { _.toString }
```

Comment:
Sorting in our case creates a lot of PrimitiveDifferences (30% - 50%) instead of OrderingDifferences.